apps/src/sites/studio/pages/levelbuilder_pixelation.js

```diff
@@ -1,19 +1,52 @@
 /** @file Javascript for the pixelation widget levelbuilder edit page */
 import $ from 'jquery';
+import 'jquery-ui';
+
+let pixelationVersionDropdown;
+let hideSlidersField;
+let initialDimensionsField;
 
 $(() => {
-  const pixelationVersionDropdown = $('#level_version');
+  pixelationVersionDropdown = $('#level_version');
+  hideSlidersField = $('.hide-sliders');
+  initialDimensionsField = $('.initial-dimensions');
 
   // When the version changes, show/hide relevant fields
   pixelationVersionDropdown.change(() => {
     showAndHideFieldsForVersion(pixelationVersionDropdown.val());
   });
 
-  // On load, show/hide fields based on version
-  showAndHideFieldsForVersion(pixelationVersionDropdown.val());
+  // On load, show/hide fields based on version without animation
+  showAndHideFieldsForVersion(pixelationVersionDropdown.val(), false);
 });
 
-function showAndHideFieldsForVersion(version) {
-  // Only show the initial width/height fields for version 1
-  $('.initial-dimensions').toggle(version === '1');
+function showAndHideFieldsForVersion(version, animate=true) {
```

Comment:
I'm fine with undoing this case, I was uncertain about it when I originally wrote it.  In this case I've written just one call where `animate` is true and one where `animate` is false, so it seems odd to have a default.  That said, I was trying to illustrate that for most _invocations_ of this method (any time the version changes) we expect to animate, and not animating is the special case (only happens on page load).  It just happened to shake out that I only had to write one of each call.

More generally I think there are good arguments for sensible defaults in API design (not least because there's precedent in the browser APIs themselves) but for a helper like this that isn't currently used much it's overkill.  I'll change it. :+1: